recipe/meta.yaml

```
@@ -49,6 +49,9 @@ requirements:
     - importlib-metadata
     - pyyaml >=3.13
     - packaging >=19.0
+    # Recommended deps
+    - scipy >=1.8
+    - matplotlib-base >=3.3,!=3.4.0,!=3.5.2
```

Could probably be simplified in pyproject.toml too but matplotlib 3.5.0 was the first version to support Python 3.10 so we're de facto not supporting 3.3 and 3.4 anymore
```suggestion
    - matplotlib-base >=3.5.0,!=3.5.2
```

---

cool, I just copied this out of pyproject.toml